solutions/1985.java

```diff
@@ -0,0 +1,15 @@
+

```

Comment:
import java.math.BigInteger;

Reply:
You will explicitly need to import this class

Reply:
Done